src/index.js

```diff
@@ -58,6 +60,8 @@ export default {
       RelationExtractorAnnotator,
       RegexNERAnnotator,
       CorefAnnotator,
+      NaturalLogicAnnotator,
+      OpenIEAnnotator,
```

Comment:
nit: please, use the same order as for the import statements (that's OCD 😆I know).